messaging/kafka-streams-reactive-messaging/src/test/resources/k8s-kafka.yaml

```diff
@@ -0,0 +1,100 @@
+apiVersion: v1
```

Comment:
This is definition for upstream based version

We need also test for product based on community version.

https://access.redhat.com/documentation/en-us/red_hat_amq/7.7/ - AMQ Streams on OpenShift
https://catalog.redhat.com/software/containers/search?p=1&product_listings_names=AMQ%20Streams
https://catalog.redhat.com/software/containers/amq7/amq-streams-kafka-25-rhel7/5ec81438dd19c77896211497 

An inspiration can be found in https://github.com/quarkus-qe/quarkus-openshift-test-suite/pull/122 (see `@OnlyIfConfigured("ts.authenticated-registry")`, ` @OnlyIfNotConfigured("ts.authenticated-registry")` parts)

I think this can be done in separate PR if you prefer, it's important to have product based coverage too.
If you need some product context, I think @Sgitario as he has some RH history already.